courses/free-courses-en.md

```diff
@@ -929,6 +929,7 @@
 
 * [Deno Beginner](https://denobeginner.com) - Ahmad Awais (email address *required*)
 * [Deno Course - Better than Node.js?](https://www.youtube.com/watch?v=TQUy8ENesGY) - The Codeholic, freeCodeCamp
+* [Deno Crash Course](https://www.youtube.com/watch?v=NHHhiqwcfRM) - Traversy Media
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
Updated the link with a playlist link for Deno.